### DISCUSSION TITLE
How do i to navigate to sub-route in the 2nd expo app?

### DISCUSSION THREAD

**Discussion by iambharath87** [expo/router#824] [Q&A]
I have a mono-repo with the following structure.

```
|_apps
|     |____ native
|                |____home
|                |____credit-card
|     |____ web
```

Both of the "native" apps are expo apps using expo-router for navigation. Here are the routes defined for both the apps:

![image](https://github.com/expo/router/assets/6663930/d658a3ad-e5ba-448e-811e-c4152b7a7df9)

The `home` app has two routes: `/` (index.tsx) and `/credit-card`.
The `credit-card` app has two routes: `/` (index.tsx) and `/details`.

The `/credit-card` screen from the home app is basically a wrapper around the `index.tsx` of the credit-card app.
I am trying to navigate from route `/` to `/credit-card/details` page but i'm getting a "Unmatched route" error.

Here is a video of this: https://user-images.githubusercontent.com/6663930/231235863-38f1acac-0029-4618-9cfe-cdb8d2568b12.mp4

Can someone please help me?

Minimal reproducible example
Here is a sample repo that shows the problem: https://github.com/iambharath87/expo-monorepo
To start the app, go to `/apps/native/home` and run `yarn ios`

**Comment by EvanBacon** [comment]
You cannot navigate between two distinctly different native apps using in-app navigation (regardless of framework). In Expo/React Native, an app is defined by having a `package.json`, there are two `package.json` files between the apps. If your intention is to combine the apps, then move the source from one `app` directory into the other. If your intention is to ship two different apps and have them navigate through the OS, then use the fully qualified URI scheme: `acme://path/to/screen`.
